### PR TITLE
Web api wiring

### PR DESCRIPTION
**Pull request by natalie-pratt**
Not finished yet - so far only have the Unsplash function going. Let me know if this is what we're looking for!

### CODE REVIEW

**Review by jk3132kv — APPROVED** [review]

**Review by MCTCJulesO — APPROVED** [review]
This looks good so far!

File: web_api/national_park/national_parks.py
Comment (technomooney) [inline web_api/national_park/national_parks.py:1]: removed the redundant code to make it a little cleaner.

File: web_api/web_api_wiring.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+def get_national_park_list(search_query):
+    """This function calls the API request function from national_parks.py with 
+    a search query entered by the user on the web app.
+    The request function either returns a response or an exception;
+    if there's an error, it will be logged, else, the create park object
+    list function is called and the list of park objects is returned."""
+    response, error = national_parks.get_parks_data(search_query)
+
+    if error:
```
Comment (technomooney) [inline web_api/web_api_wiring.py:28]: we should put a return None there so it still returns something if there is an error.

File: web_api/web_api_wiring.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+def get_weather_forecast_object(lat, lon):
+    """This function calls the API request function from open_weather_map.py with 
+    the latitude and longitude of the park entered by the user on the web app.
+    The request function either returns a response or an exception;
+    if there's an error, it will be logged, else, the extract_data
+    function is called and the forecast dictionary is returned."""
+    response, error = open_weather_map.get_api_response(lat, lon)
+
+    if error:
+        print(f'An error has occurred: {error}') # TODO - switch to log instead of print
```
Comment (technomooney) [inline web_api/web_api_wiring.py:45]: return none here too

**Review by technomooney — CHANGES_REQUESTED** [review]
Just a few changes from what i can see

**Comment by natalie-pratt** [comment]
@technomooney I made the changes

**Review by technomooney — APPROVED** [review]
ok looks good. if i find issues i will fix them in the ui_finalization branch. only cause thats where the issues are likely to come to light.

**Comment by natalie-pratt** [comment]
Completes issue #22

**Pull request merged**